COMPILING.md

```diff
@@ -300,6 +300,21 @@ the need to integrate JBMC as a separate project. Be aware that you need to
 change the build location (Select project in Eclipse -> Properties -> C/C++ 
 Build) to one of the src directories.
 
+# WORKING WITH DOCKER
+
+To compile and run the tools in a Docker container, do the following:
+
+1. From the root folder of the project, run `$ docker build -t cbmc .`
+2. After the building phase has finished, there should be a new 
+   image with the CProver binaries installed under `/usr/bin/`.
+
+   To start a container using that image as a base, run `$ docker run -i -t cbmc`
+   This will result in dropping you to a new terminal inside the container. To
+   load files for analysis into the container, one way is by mounting the folder
+   that contains the tests to the container. A possible invocation that does that
+   is: `$ docker run -v local/path/with/files:/mnt/analysis -i t cbmc`. In the
```

Comment:
The Docker documentation strongly suggests to avoid using `-v` in favour of the slightly different `--mount` [here](https://docs.docker.com/storage/bind-mounts/)

> **Tip:** New users should use the --mount syntax. Experienced users may be more familiar with the -v or --volume syntax, but are encouraged to use --mount, because research has shown it to be easier to use.

Reply:
I don't think it's easier than `-v`, especially for such a simple use case.

Apart from that, this is indicative usage, and not a docker tutorial. If a user prefers a different invocation, he can use that. Right now this is massively preferred by me, as it's much simpler than the `--mount` equivalent.

Reply:
If the `--mount` syntax is easier to read then I am all for switching it over. As this is, it isn't necessarily clear which path is in the container and which is outside, without consulting the docker manual.